pkg/stream/multi_producer.go

```diff
@@ -0,0 +1,31 @@
+package stream
+
+import (
+	"context"
+)
+
+type multiProducer []Producer
+
+func NewMultiProducer(producers ...Producer) Producer {
+	return multiProducer(producers)
+}
+
+func (mp multiProducer) WriteOne(ctx context.Context, model interface{}, attributeSets ...map[string]interface{}) error {
+	for _, p := range mp {
+		err := p.WriteOne(ctx, model, attributeSets...)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
```

Comment:
please always leave an empty line before a return except it is the only statement in a block

Reply:
Will do